packages/twenty-front/src/modules/ui/layout/tab/components/TabListDropdown.tsx

```diff
@@ -0,0 +1,77 @@
+import { Dropdown } from '@/ui/layout/dropdown/components/Dropdown';
+import { DropdownContent } from '@/ui/layout/dropdown/components/DropdownContent';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { TabAvatar } from '@/ui/layout/tab/components/TabAvatar';
+import { SingleTabProps } from '@/ui/layout/tab/components/TabList';
+import { TabMoreButton } from '@/ui/layout/tab/components/TabMoreButton';
+import { MenuItemSelectAvatar } from 'twenty-ui/navigation';
+
+type TabListDropdownProps = {
+  dropdownId: string;
+  onClose: () => void;
+  overflow: {
+    hiddenTabsCount: number;
+    isActiveTabHidden: boolean;
+    firstHiddenTabIndex: number;
+  };
+  tabs: {
+    visible: SingleTabProps[];
```

Comment:
I'm a bit confused by this API:
hiddenTabs : tabs.visible.slice(...)

what does visible mean actually?
then it's a bit weird to have it grouped under tabs. Why not just: tabs: SingleTabProps[] and activeTabId as props